recipes/newmat/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,221 @@
+/// \ingroup newmat
```

Comment:
Please, simplify this file, use a single method like printing library version.

As documented (https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/test_packages.md#minimalist-source-code) the goal is testing the generated package, not running an entire example. Remind that once this code is merged, CCI will become the maintainer, so less code is always better.

Reply:
fixed in 2aea9744eebd36f4c03fdc26fc83efec0933766a